lib/oli_web/live/common/utils.ex

```diff
@@ -35,6 +35,32 @@ defmodule OliWeb.Common.Utils do
     date(Map.get(item, attr_name), opts)
   end
 
+  @doc """
+    Rounds up a grading score to two significant figures.
+    For numbers with no decimals, or non-significant zeros after the comma, it keeps only one zero
+
+    ## Examples
+    iex> format_score(200.0)
+    200.0
+
+    iex> format_score(120.2333)
+    120.23
+
+    iex> format_score(88.00)
+    88.0
+
+    iex> format_score(78.479)
+    78.48
+
+    iex> format_score(0.0)
+    0.0
+  """
+  @spec format_score(float) :: float
+  def format_score(score) when is_float(score) do
+    formatted_score = Float.round(score, 2)
+    if score - formatted_score != 0.0, do: formatted_score, else: score
```

Comment:
I might be missing something, but is this line necessary? If `score - formatted_score` is `0` it means they are equal, so you can just return always the `formatted_score`